buildscripts/observability-test/Dockerfile

```diff
@@ -17,11 +17,18 @@
 # Stage 1: Build the interop test client and server
 #
 
-FROM openjdk:11.0.16-jdk-slim-bullseye AS build
+FROM eclipse-temurin:11-jdk AS build
 
 WORKDIR /grpc-java
 COPY . .
 
+# Intentionally after the app COPY to force the update on each build.
+# Update Ubuntu system packages:
```

Comment:
Yes - this needs to be moved to the second stage of the build.

Reply:
Done. Thanks for the comment.